### DISCUSSION TITLE
support Pagination with unknown boundries

### DISCUSSION THREAD

**Discussion by guynaa** [mantinedev/mantine#2155] [Feature requests]
i would like to use https://mantine.dev/core/pagination/
but my use case is the following:
at each moment i know if i have a "next" page or a "prev" page,
so i basically need just the 2 prev/next buttons that might be disabled.

is it possible to configure the object in this way?

**Comment by rtivital** [comment]
Hi @guynaa, you do not need pagination for that, use ActionIcon component instead.

**Reply by guynaa** [reply]
how can i style them in the same way the current pagination buttons look (icon, size, etc,..)

**Reply by rtivital** [reply]
`<ActionIcon variant="default"><YourIcon /></ActionIcon>`
